config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  root to: 'blogs#index'
+  resources :blogs
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
```

Comment:
Please go through project [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md) and make sure every requirement is met.

Reply:
Root root added